cmd/dataweavetransformation-adapter/Dockerfile

```diff
@@ -14,10 +28,14 @@ RUN go build -o /dataweavetransformation-adapter ./cmd/dataweavetransformation-a
 
 FROM debian:stable-slim
 
+# Ensure the /kodata entries are present
+COPY --from=builder /go/triggermesh/.git/ /kodata/
```

Comment:
Do we really need the contents of the entire `.git` directory? In the example at https://github.com/google/ko#static-assets I see they only the symlink `.git/HEAD` file, so will just copying over that one file suffice?

Reply:
Only a few selected files are copied in practice, due to this exclusion list:

https://github.com/triggermesh/triggermesh/blob/ba3b7ab1642070cc6a6e804306faf7463e7738f7/.dockerignore#L19

Reply:
ah ha! thanks for the clarification